### ISSUE TITLE
[Feature] Insert .csv files as table

### ISSUE THREAD

**Issue by K-lone** [typora/typora-issues#5768]
Hi everybody,

first of all, thanks for this amazing tool ! It would be marvelous to be able to insert .csv files in Typora. In order to generate and change the tables in other sofware and be able to include or display them in a .md document in Typora.

Bye !

**Comment by ZJsheep** [comment]
Same suggestion. Actually there are many tools on Internet which can convert csv to markdown tables, but it would be great if typora support this feature locally :)

**Comment by mikedeskevich** [comment]
Yes, I might suggest a more general "include" that includes the text of another file to be interpreted in the current file.  I use the gnu m4 macro processor for that right now, where I write a bunch of .md files and then concatenate them into a bigger md file for publication as a pdf or something.  I'd love to do all of that in Typora.  

And with that, interpreting csvs as tables would be huge.  Right now, I often write a report in .md and the images that are included are generated by a python script.  So if I get new data or do a new analysis, I rerun the script and the images are automatically updated.  If I could do that with tables or other chunks of data (even mermaid code).  I could have a dynamic workflow.
